driver/include/sensor.h

```diff
@@ -82,8 +85,8 @@ typedef enum {
     FRAMESIZE_96X96,    // 96x96
     FRAMESIZE_QQVGA,    // 160x120
     FRAMESIZE_QCIF,     // 176x144
+    FRAMESIZE_200X200,  // 200x200
```

Comment:
seems you have one more place left here and somehow removed 240x240? Could you please fix this?

Reply:
This error has been fixed.